packages/eslint-config-eslint/base.js

```diff
@@ -68,7 +68,6 @@ const jsConfigs = [js.configs.recommended, {
         "no-caller": "error",
         "no-confusing-arrow": "error",
         "no-console": "error",
-        "no-constant-binary-expression": "error",
```

Comment:
This rule is now in the `recommended` config so no need to enable it here again.